lib/ansible/modules/cloud/amazon/ec2_vpc_nat_gateway_facts.py

```diff
@@ -107,7 +108,16 @@ def get_nat_gateways(client, module, nat_gateway_id=None):
     except Exception as e:
         module.fail_json(msg=str(e.message))
 
-    return [camel_dict_to_snake_dict(gateway) for gateway in result['NatGateways']]
+    for gateway in result['NatGateways']:
+        # Turn the boto3 result into ansible_friendly_snaked_names
+        converted_gateway = camel_dict_to_snake_dict(gateway)
+        if 'tags' in converted_gateway:
+            # Turn the boto3 result into ansible friendly tag dictionary
+            converted_gateway['tags'] = boto3_tag_list_to_ansible_dict(converted_gateway['tags'], 'key', 'value')
```

Comment:
Shouldn't need the second and third parameters now `boto3_tag_list_to_ansible_dict` has been fixed.

Not a blocker though.

Reply:
Ah I see, no problems. I'll update shortly